src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -146,12 +148,17 @@ export function computeAlterationTypes(alterationData: ICaseAlteration[]): strin
 }
 
 export function getPseudoOqlSummaryByAlterationTypes(oqlData: {[oqlLine: string]: IOqlData}, oqlLine: string, alterationTypes: string[]) : PseudoOqlSummary {
-    const alteratedPseudoOqlSummarys = _.chain(alterationTypes)
-        .map(type => generatePseudoOqlSummary(oqlData, oqlLine, type))
-        .filter((summary) => summary ? summary.summaryClass === styles.alterationSpan : false)
-        .value();
+    const pseudoOqlSummarys = _.map(alterationTypes, (type) => generatePseudoOqlSummary(oqlData, oqlLine, type));
+    const notProfiledPseudoOqlSummarys = _.filter(pseudoOqlSummarys, (summary) => summary ? summary.summaryClass === styles.notProfiledSpan : false);
```

Comment:
where does styles come from here?

Reply:
if this is a css class name (and styles is imported stylesheet), we shouldn't be using a class name like this for logical purposes.  needs to be more abstract.